### PR TITLE
Add missing lifetime annotation required by Rust 1.64.0

### PR DESCRIPTION
**Pull request by fusillicode**

### CODE REVIEW

File: src/esrs/aggregate.rs [hunk 1]
```diff
@@ -106,7 +106,7 @@ pub trait AggregateManager: Aggregate {
     /// the aggregate state by order of their sequence number
     ///
     /// You should _avoid_ implementing this function, and be _very_ careful if you decide to do so.
-    async fn load(&self, aggregate_id: impl Into<Uuid> + Send) -> Option<AggregateState<Self::State>> {
+    async fn load(&self, aggregate_id: impl Into<Uuid> + Send + 'async_trait) -> Option<AggregateState<Self::State>> {
```
Comment (cottinisimone) [inline src/esrs/aggregate.rs:109]: Don't get why is this needed. I'm using rust 1.64.0 and no one complains for this

Comment (fusillicode) [reply src/esrs/aggregate.rs:109]: Uhm, that's strange 🤔 

I'll try to deep dive as soon as possible 🙏

Comment (fusillicode) [reply src/esrs/aggregate.rs:109]: Local "cache" issue. No esrs issue, my bad 🤦‍♂️
